common/src/main/java/tachyon/util/network/NetworkAddressUtils.java

```diff
@@ -57,8 +57,7 @@ public static String getLocalHostName(TachyonConf conf) {
     if (sLocalHost != null) {
       return sLocalHost;
     }
-    int hostResolutionTimeout = conf.getInt(Constants.HOST_RESOLUTION_TIMEOUT_MS,
-        Constants.DEFAULT_HOST_RESOLUTION_TIMEOUT_MS);
```

Comment:
Yes, there are currently a bunch of unused DEFAULT values in `Constants`, which I will remove in separate PR.